recipes/ff/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "ff" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+    url: https://github.com/hep-packaging-coordination/ff/archive/v{{ version }}.tar.gz
+    sha256: 2329b3ca147825b660e07dac45ed85d397a1bea2f6b886518eb4d09900da5503
+    patches:
+      # Can be removed after the patched QCDLoop version is merged in v2.0.1.
+      - update-makefile.patch
+      # c.f. https://github.com/hep-packaging-coordination/ff/pull/8
+      - allow-longer-paths-for-ffopen.patch
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ name }}-static
+
+    build:
+      skip: true  # [win or (osx and x86)]
```

Comment:
Things work for arm64 osx but not x86, so skipping x86 for the time being, but not using `# [not linux]` to allow for rerenders to add additional builds.